specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -396,6 +396,206 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}/privateDnsZoneGroups/{privateDnsZoneGroupName}": {
+      "delete": {
+        "tags": [
+          "PrivateDnsZoneGroup"
+        ],
+        "operationId": "PrivateDnsZoneGroups_Delete",
+        "description": "Deletes the specified private dns zone group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "privateEndpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private endpoint."
+          },
+          {
+            "name": "privateDnsZoneGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private dns zone group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "200": {
+            "description": "Delete successful."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "Delete private dns zone group": {
+            "$ref": "./examples/PrivateEndpointDnsZoneGroupDelete.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "PrivateDnsZoneGroup"
+        ],
+        "operationId": "PrivateDnsZoneGroups_Get",
+        "description": "Gets the private dns zone group by specified private endzone.",
```

Comment:
why is this needed?

Reply:
The GET operation? Isn't that part of the CRUD operation functionality we wanted?

Reply:
Or do you mean the description, would it be more accurate to say, "Gets the private dns zone group resource by specified private dns zone group name."?

Reply:
Besides that, I refered to the privateendpoint API for operationId, description, and tag setup. This link also mentions putting in the operationId for generation purposes. https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md

Reply:
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md provides better explanation for this, "...Tags can be used for logical grouping of operations by resources or any other qualifier." It's meant I think just to be an easier identifier for documentation